### ISSUE TITLE
New Grafana unified alerting

### ISSUE THREAD

**Issue by Puneeth-n** [weaveworks/grafanalib#445]
Should the lib be somehow updated to support the new unified alerting?

https://grafana.com/docs/grafana/latest/alerting/unified-alerting/

**Comment by JamesGibo** [comment]
Yeah, would be nice to have and satanize some of the old alerting support.

**Comment by vcandeau** [comment]
I think it's the purpose of the pull request i have made to add AlertRuler

**Comment by OscarVanL** [comment]
New Grafana alerting is now supported, maybe this can be closed?
